src/main/java/org/prebid/server/bidder/model/BidderInfoSeatBid.java

```diff
@@ -0,0 +1,49 @@
+package org.prebid.server.bidder.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import org.prebid.server.auction.model.BidInfo;
+import org.prebid.server.bidder.Bidder;
+import org.prebid.server.proto.openrtb.ext.response.ExtHttpCall;
+
+import java.util.List;
+
+/**
+ * Seatid returned by a {@link Bidder}.
+ * <p>
+ * This is distinct from the {@link com.iab.openrtb.response.SeatBid} so that the prebid-server ext can be passed
+ * back with type safety.
+ */
+@AllArgsConstructor(staticName = "of")
+@Value
+public class BidderInfoSeatBid {
+
+    /**
+     * List of bids which bidder wishes to make.
```

Comment:
Probably we can skip java docs in this model since we've already described in original `BidderSeatBid`.